_topic_map.yml

```diff
@@ -654,6 +654,10 @@ Topics:
     File: editing-egress-firewall
   - Name: Removing an egress firewall from a project
     File: removing-egress-firewall
+  - Name: Configuring an egress IP address
```

Comment:
There are no explicit steps in this assembly. Is this intentional? If so, you need a new title.

Reply:
@kalexand-rh, forgot to add the labeling procedure.